helm/cosmo/charts/router/values.yaml

```diff
@@ -211,3 +211,8 @@ configuration:
   # -- The path to the router execution config file. Before, you have to mount the file as a volume and set the path here.
   # -- A possible to solution could be to use an init container to download the file from a CDN. If set, polling of the config is disabled.
   routerConfigPath: ""
+
+  prometheus:
```

Comment:
Don't forget to document each property and run `make docs` in the helm repository.

Reply:
Please let me know if it is fixed now